src/internal/compilerSupport/v2.ts

```diff
@@ -204,7 +216,7 @@ export function callFunction(rawFnArgs: Array<any>, def: FunctionCallDef, script
     }
 
     const promise = new Promise((resolve, reject) => {
-        const particle = Particle.createNew(script, config?.ttl);
+        const particle = Particle.createNew(script, config?.ttl || undefined);
```

Comment:
Why do you need `|| undefined` here? 

`config?.ttl` will return `number | undefined`

Reply:
It's done to treat 0 as if the option is not set.
I want to make the TTL behaviour consistent with the option in FluencePeer config.

Reply:
Why do you want to treat `0` as if the option is not set?

Reply:
If a user'd want it to be unset, he'd set `undefined`. If she'd want it to be `0`, then let her do it.

Reply:
Timeout of 0 is pointless IMO. Why would anyone want to call a function which will fail right away?

Reply:
If you don't see a use-case right away, it doesn't mean there's no. I suggest not overthinking and just passing values without changing their meaning in implicit ways.